sdk/metric/EXPERIMENTAL.md

```diff
@@ -40,6 +40,28 @@ Disable the cardinality limit.
 unset OTEL_GO_X_CARDINALITY_LIMIT
 ```
 
+### Exemplars
```

Comment:
I think we are missing documentation for  `OTEL_METRICS_EXEMPLAR_FILTER`.

Reply:
Updated.